src/windows.rs

```diff
@@ -204,19 +206,37 @@ unsafe fn wstr_to_os_string(wstr: PWSTR) -> OsString {
     OsString::from_wide(wstr_slice)
 }
 
-unsafe fn get_date_deleted_unix(item: &IShellItem2) -> Result<i64, Error> {
+unsafe fn get_date_deleted_fs(item: &IShellItem2) -> Result<u64, Error> {
+    let time = item.GetFileTime(&SCID_DATE_DELETED)?;
+    let time_u64 = ((time.dwHighDateTime as u64) << 32) | (time.dwLowDateTime as u64);
+
+    Ok(time_u64)
+}
+
+fn date_fs_to_unix(date: &u64) -> Result<i64, Error> {
```

Comment:
`date` should not be a reference.

Reply:
I'm sorry, I'm at my very beginning of my rust and git/github journey, so I'm not really sure of what I am doing.